src/sentry/models/sentryfunction.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+
+from sentry.db.models import BaseManager, DefaultFieldsModel, FlexibleForeignKey
+
+
+class SentryFunctionManager(BaseManager):
```

Comment:
@vpandiarajan20 no lol I meant remove the manager, you can use the `BaseManager` instead. We don't need a custom manager.

Reply:
oh gotcha!

Reply:
Removed for now, but we'll need a custom manager at some point right?

Reply:
@vpandiarajan20 For what purpose? I don't think we do.